plugins/woocommerce/client/blocks/assets/js/blocks/product-collection/edit/ProductPicker.tsx

```diff
@@ -52,36 +48,34 @@ const ProductPicker = (
 		  );
 
 	return (
-		<div { ...blockProps }>
```

Comment:
This one as well. Basically in the  https://github.com/woocommerce/woocommerce/blob/f861f01d3c9cc1c0d632bbb4e651f829654c41d7/plugins/woocommerce/client/blocks/assets/js/blocks/product-collection/edit/index.tsx you can see there's a choice which component is rendered and any of them can be a "main" block and each of them should get block props.